docs/source-pytorch/fabric/fabric.rst

```diff
@@ -3,125 +3,100 @@ Fabric
 ######
 
 
-:class:`~lightning_fabric.fabric.Fabric` enables pure PyTorch users to scale their existing code
-on any kind of device while retaining full control over their own loops and optimization logic.
+:class:`~lightning_fabric.fabric.Fabric` enables PyTorch users to accelerate and scale their models while retaining full control over their training loop and optimization logic.
 
-.. image:: https://pl-public-data.s3.amazonaws.com/docs/static/images/lite/lightning_lite.gif
-    :alt: Animation showing how to convert your PyTorch code to Fabric.
-    :width: 500
-    :align: center
+.. code-block:: diff
```

Comment:
TIL! Does this add color-coding for additions and deletions?

Reply:
Yes, it looks like this:

<img width="1233" alt="image" src="https://user-images.githubusercontent.com/5495193/210539756-995aabac-2ddc-432b-9d0a-82010a3920b5.png">